Sources/SwifterSwift/UIKit/UIImageExtensions.swift

```diff
@@ -336,18 +336,23 @@ public extension UIImage {
 public extension UIImage {
     /// SwifterSwift: Create UIImage from color and size.
     ///
+    ///     let redImage = UIImage(color: .red)
+    ///     let blueImage = UIImage(color: .blue, size: CGSize(width: 20, height: 20))
+    ///
     /// - Parameters:
     ///   - color: image fill color.
     ///   - size: image size.
-    convenience init(color: UIColor, size: CGSize) {
-        UIGraphicsBeginImageContextWithOptions(size, false, 1)
+    convenience init(color: UIColor, size: CGSize = CGSize(width: 1, height: 1)) {
```

Comment:
Can you provide an use case where is useful to have 1x1 default size? Personally I would prefer not having a default size value.

Reply:
In some scenarios, there is no difference between providing a 10x10 size and a 1x1 size：
```swift
func fetchImageFromUrl(url: URL) -> UIImage {
    // ...

    // If download image failure
    return UIImage(color: .gray, size: CGSize(width: 10, height: 10))
}
```

Or just set the background color for the different states of the button：
```swift
let button = UIButton()
button.setImage(UIImage.init(color: .blue), for: .normal)
button.setImage(UIImage.init(color: .gray), for: .disabled)
```

Also, adding a default parameter to a method doesn't mean you can't actively provide that parameter, it's just one more option. You can still use your preferred method:
```Swift
let redImage = UIImage(color: .red)
let blueImage = UIImage(color: .blue, size: CGSize(width: 20, height: 20))
```

Reply:
> Also, adding a default parameter to a method doesn't mean you can't actively provide that parameter, it's just one more option.

Sure, my question is more in which case a user would want to use a default value that would return a 1x1 image? To me doesn't make much sense a default so specific. Normally APIs provide size values as default it has to be something more general that would apply to most default cases which to me it is not the case for a 1x1 default.

Reply:
From a user perspective imagine calling a `UIImage(color: .red)` and trying to reason why the API provider decided a 1x1 image should be default. Default values should feel naturally expected when defining an API in my opinion. 
That said, I see the example of the button background, but still think a 1x1 default is not the right path.
But is only my opinion, let's wait for others thoughts as well to see what they think =]